gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -81,14 +126,16 @@ def run(
 
         # convert translation direction in global frame using rotations.
         w_i2Ui1_measurements = BinaryMeasurementsUnit3()
+        valid_i2_i1 = []
```

Comment:
Can we rename to something more expressive? `nonnull_global_rotations_i2i1_indices`? Or is it too long?

Reply:
I think maybe a bit too long

Reply:
Its too long :O 
i1_i2 -> means indices, just like i2Ri1 means a rotation (also not very expressive IMO). I dont mind changing it, but I cannot think of a shorter and better name. Besides, it might be removed once we fix the gtsam issue I mentioned in another comment.